data_model/src/expression.rs

```diff
@@ -65,6 +62,21 @@ impl<V: TryFrom<Value>> EvaluatesTo<V> {
         self.expression.len()
     }
 
+    /// Construct new [`EvaluatesTo`] from [`ExpressionBox`] without type checking.
+    ///
+    /// # Warning
+    /// Prefer using [`Into`] conversions rather than this method,
+    /// cause it does not check type at compile-time.
+    ///
+    /// Exists mainly to test unsupported expressions.
```

Comment:
```suggestion
    /// Prefer using [`Into`] conversions rather than this method,
    /// because it does not check the value type at compile-time.
    ///
    /// This method is mainly to test unsupported expressions.
```